src/main/topLevelPages/myDashboard/newVirtualContributorWizard/CreateNewVirtualContributor.tsx

```diff
@@ -84,7 +87,10 @@ const CreateNewVirtualContributor = ({
   onUseExistingKnowledge,
   onUseExternal,
   loading,
+  getVisual,
 }: CreateNewVirtualContributorProps) => {
+  const [vcAvatar, setVcAvatar] = useState<VisualWithAltText>();
```

Comment:
why keeping a state here, if the only thing you want from it is to push it up to the parent?